src/app/Plans/components/Wizard/PlanWizard.tsx

```diff
@@ -121,16 +114,23 @@ const PlanWizard: React.FunctionComponent<IPlanWizardProps> = ({
   };
 
   return (
-    <Modal isOpen width="95%" showClose={false} hasNoBodyWrapper>
+    <PageSection variant="light">
+      <Breadcrumb className={`${spacing.mbLg} ${spacing.prLg}`}>
+        <BreadcrumbItem>
+          <Link to={`/plans`}>Migration plans</Link>
+        </BreadcrumbItem>
+        <BreadcrumbItem>Plan Wizard</BreadcrumbItem>
+      </Breadcrumb>
       <Wizard
         title="Create a Migration Plan"
```

Comment:
Let's remove the `title` prop and instead add a title to the page section above, by the breadcrumb bar like we have on our other pages. The black title bar looks a little awkward embedded on the page.